test/unit/browser/new-browser.js

```diff
@@ -202,6 +202,56 @@ describe('browser/new-browser', () => {
                 return assert.isRejected(launchBrowser());
             });
         });
+
+        describe('on wd init fail', () => {
+            it('should fail launch browser', () => {
+                wd.init.returns(Promise.reject(new Error('o.O')));
+                return assert.isRejected(launchBrowser());
+            });
+
+            it('should fail with GeminiError and empty Reason in message', () => {
+                wd.init.returns(Promise.reject({data: '', message: ''}));
+
+                return launchBrowser()
+                    .catch((e) => {
+                        assert.notInclude(e.message, 'Reason:');
```

Comment:
тут можно в одну строку написать. И я бы тут проверял, что `e.message` не изменился. Вдруг ты завтра захочешь вместо строки 'Reason: ...' писать 'Because: ...' например. Придется в двух местах менять.